pkg/controller/kieapp/defaults/defaults_test.go

```diff
@@ -2240,6 +2229,20 @@ func checkLabels(t *testing.T, labels map[string]string, component, version stri
 	assert.Equal(t, "Red_Hat", labels[constants.LabelRHcompany])
 }
 
+func checkClusterLabels(t *testing.T, cr *api.KieApp, object api.CustomObject) {
+	if cr.Spec.Version == constants.CurrentVersion {
+
+		for _, dc := range object.DeploymentConfigs {
+			assert.NotNil(t, dc.Spec.Template.Labels[constants.ClusterLabel])
+			assert.True(t, strings.HasPrefix(dc.Spec.Template.Labels[constants.ClusterLabel], constants.ClusterLabelPrefix))
+		}
+		for _, ss := range object.StatefulSets {
+			assert.NotNil(t, ss.Spec.Template.Labels[constants.ClusterLabel])
+			assert.True(t, strings.HasPrefix(ss.Spec.Template.Labels[constants.ClusterLabel], constants.ClusterLabelPrefix))
+		}
+	}
+}
+
```

Comment:
this is a good start, but we need to know that all of the different console and kie variations are set correctly. instead of assert.True, maybe assert.Equal and check for the full `jgrp.k8s.[[.ApplicationName]].[[.Console.Name]]` variations... kieserver and dashbuilder as well.

Reply:
ok